sdk/keyvault/Azure.Security.KeyVault.Keys/samples/Sample3_GetKeys.cs

```diff
@@ -82,12 +82,17 @@ public void GetKeysSync()
 
             // The Cloud RSA Key and the Cloud EC Key are no longer needed.
             // You need to delete them from the Key Vault.
-            client.DeleteKey(rsaKeyName);
-            client.DeleteKey(ecKeyName);
+            DeleteKeyOperation rsaKeyOperation = client.StartDeleteKey(rsaKeyName);
+            DeleteKeyOperation ecKeyOperation = client.StartDeleteKey(ecKeyName);
 
             // To ensure secrets are deleted on server side.
```

Comment:
We should mention that this is only necessary if you intend to purge the deleted key. I don't want users to think this is always needed since it can take quite a bit of time.